tests/bootstrap.php

```diff
@@ -7,6 +7,8 @@
 
 require_once $_tests_dir . '/includes/functions.php';
 
+remove_action( 'init', array( 'VIP_Cache_Manager', 'init' ) );
```

Comment:
Tests failing because this is being called too early.
```
Error: Call to undefined function remove_action() in /home/travis/build/Automattic/vip-go-mu-plugins/tests/bootstrap.php on line 10
```